msdk-io/msdk-io-mzml2/pom.xml

```diff
@@ -0,0 +1,42 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>io.github.msdk</groupId>
+		<artifactId>msdk-io</artifactId>
+		<version>0.0.8-SNAPSHOT</version>
+	</parent>
+	<artifactId>msdk-io-mzml2</artifactId>
+
+	<repositories>
+        <repository>
+            <!-- EBI repository -->
+			<id>nexus-ebi-release-repo</id>
+			<name>The EBI Maven repository</name>
+			<url>http://www.ebi.ac.uk/Tools/maven/repos/content/groups/ebi-repo/</url>
+        </repository>
+	</repositories>
+
+	<dependencies>
+
+		<dependency>
+			<groupId>io.github.msdk</groupId>
+			<artifactId>msdk-datamodel</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		
+		<dependency>
+			<groupId>io.github.msdk</groupId>
+			<artifactId>msdk-spectra-spectrumtypedetection</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+
+		<dependency>
+			<groupId>uk.ac.ebi.jmzml</groupId>
+			<artifactId>jmzml</artifactId>
```

Comment:
the jmzml dependency is not needed anymore, because your module will do the parsing